content/hardware/build_instructions.md

```diff
@@ -501,15 +496,15 @@ In this section, we will configure the buck converter. The buck converter steps
  
 **Buck Converter Voltage Adjustment**
 
-The following instructions detail voltage adjustment for the **DZS Elec LM2596** buck converter. If you are using a different buck converter, follow the manufacturer's instructions to set the output voltage to 5V.
+The following instructions detail voltage adjustment for the **DZS Elec LM2596** buck converter. If you are using a different buck converter, follow the manufacturer's instructions to set the output voltage to 5V. (or 14V if using the Xavier NX)
```

Comment:
The way it is written, the fact that it should instead be set to 14V if using the NX could be easily missed. Make it much more verbose as to what voltage should be set for each configuration (because bad things could happen if they mess it up).